opm/parser/eclipse/CMakeLists.txt

```diff
@@ -271,7 +272,7 @@ EclipseState/Tables/RtempvdTable.hpp
 EclipseState/Tables/OilvisctTable.hpp
 EclipseState/Tables/GasvisctTable.hpp
 EclipseState/Tables/WatvisctTable.hpp
-EclipseState/Tables/PvtgOuterTable.hpp
+EclipseState/Tables/PvtgTable.hpp
```

Comment:
but currently there is no `PvtoTable` class anymore? (I might have missed something, though.)

also, I think that these two tables should continue to exhibit different names: the keywords behave differently (the PVTG keyword tabulates (p, Rv) while PVTO tabulates (Rs, p) IIRC) and trying to "over-generalize" usually leads to quite opaque code. (note that even if the keyword behaviour would be functionally the same I'd prefer to have them separate because just because it *is* easily possible to generalize, it is not always a good idea.)